SSMP/Networking/Server/ServerConnectionManager.cs

```diff
@@ -102,6 +104,15 @@ public void ProcessClientInfo(ClientInfo clientInfo) {
             Logger.Error($"Exception occurred while executing the connection request event:\n{e}");
         }
 
+        SendServerInfo(serverInfo);
+        
+        return serverInfo;
+    }
+
+    /// <summary>
+    /// Sends the ServerInfo packet to the client.
+    /// </summary>
+    public void SendServerInfo(ServerInfo serverInfo) {
```

Comment:
According to the comment on `ProcessClientInfo`, the `ServerInfo` should't be send by callers of the method. So perhaps this should be private then?

```suggestion
    private void SendServerInfo(ServerInfo serverInfo) {
```